pylint/checkers/base.py

```diff
@@ -2098,6 +2162,52 @@ def _name_became_keyword_in_version(name, rules):
                 return ".".join(str(v) for v in version)
         return None
 
+    @staticmethod
+    def _assigns_typevar(node: Optional[nodes.NodeNG]) -> bool:
+        """Check if a node is assigning a TypeVar"""
+        if isinstance(node, astroid.Call):
```

Comment:
```suggestion
        if not isinstance(node, astroid.Call):
            return False
```
Would permit to remove the return False at the end ?

Reply:
This creates a `inconsistent-return-statements` warning.

Reply:
This needs the other suggestion that made mypy fails, nevermind.